sos/plugins/logs.py

```diff
@@ -8,10 +8,11 @@
 
 import os
 import glob
-from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+from sos.plugins import (Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin,
+    CosPlugin)
 
 
-class Logs(Plugin):
+class Logs(Plugin, CosPlugin):
```

Comment:
Ditto here: it's hard to believe that you don't need any of the log path adjustments that the other derived classes in this file use.

Reply:
Oh sorry for the confusion. I changed this into a separate class in the following commit.